README.md

```diff
@@ -47,6 +47,17 @@ Options:
   -h, --help         output usage information
 ```
 
+## Deploying on a Raspberry Pi
+
+```
+sudo apt-get install git
+sudo setcap cap_net_bind_service=+ep `readlink -f \`which node\``
```

Comment:
What does this magical line do?

Reply:
https://serverfault.com/questions/665709/allowing-node-js-applications-to-run-on-port-80

Reply:
It's a cheap way to give a programme privileged port access.

Planning to use systemd/launchd mechanisms instead. They can listen on the port as PID 1 and pass the socket to our process as a FD. Haven't got that working yet.